apps/settings/js/panels/simpin/simpin.js

```diff
@@ -126,15 +127,17 @@ define(function(require) {
       cardIndex = parseInt(cardIndex, 10);
 
       switch (type) {
-        case 'checkSimPin':
```

Comment:
why not keep this switch condition, remove `simPinSwitched` and handle with `this._elements.simPinContainer.addEventListener('change', this)` ?